rseng/main/taxonomy/taxonomy-1.0.0.yaml

```diff
@@ -54,26 +54,10 @@
       uid: RSE-taxonomy-provenance-metadata-tools
       color: mediumseagreen
       example: schema.org, codemeta, etc.
-  - name: Used for research but not explicitly for it
-    uid: RSE-taxonomy-used-not-explicitly-for
-    color: darkslateblue
-    children: 
-    - name: Databases
-      uid: RSE-taxonomy-databases
-      example: container registry, software database, data text files, etc.
-      color: darkkhaki
-    - name: Application Programming Interfaces
-      uid: RSE-taxonomy-application-programming-interfaces
-      color: darkturquoise
-      example: Pubmed, arXiv, Orcid, etc.
-    - name: Frameworks
-      uid: RSE-taxonomy-frameworks
-      example: documentation generators, content management systems, django
-      color: darkseagreen
-  - name: Incidentally used for research
+  - name: Used to manage infrastructure
     uid: RSE-taxonomy-incidentally-used-for-research
```

Comment:
Make sure unique identifiers are matching the names here!

Reply:
This is an interesting choice of top level category because arguably any family of tools could be used to manage infrastructure, but I'm not sure it's a definitive category, if that makes sense?

Reply:
I guess we have a divergence on what we call infrastructure then. For me it’s anything that describe the machine / network / ... infrastructure we use. LC deals with infrastructure code, CASC deals with scientific code.
I understand that you have a broader view than mine of this, so if that does not makes sense to you, so be it.

Reply:
It's definitely an overloaded term, agree!

For now let's focus on the underlying categories - are there any changes that you'd want to see for our survey? If not, let's just focus on tweaking the examples for the survey (and we won't need changes here after all!)